packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -220,6 +232,68 @@ const Breakouts = WebexPlugin.extend({
 
     this.breakouts.set(Object.values(breakouts));
   },
+
+  /**
+   * Make enable breakout resource
+   * @returns {Promise}
+   */
+  enableBreakouts() {
+    if (this.breakoutServiceUrl) {
+      // @ts-ignore
+      return this.webex
+        .request({
+          method: HTTP_VERBS.POST,
+          uri: this.breakoutServiceUrl,
+          body: {
+            locusUrl: this.locusUrl,
+          },
+        })
+        .catch((err) => {
+          LoggerProxy.logger.error(
+            `Meeting:request#touchBreakout --> Error provisioning error ${err}`
+          );
+          throw err;
+        });
+    }
+
+    return Promise.reject(new Error(`enableBreakouts: the breakoutServiceUrl is empty`));
+  },
+
+  /**
+   * Make the meeting enbale or disable breakout session
+   * @param {boolean} enable
+   * @returns {Promise}
+   */
+  async toggleBreakout(enable) {
+    if (this.enableBreakoutSession === undefined) {
+      await this.enableBreakouts().then((response) => {
+        // enableBreakouts default return enableBreakoutSession:true
+        if (!enable) {
+          // if enable is false, updateBreakout set the param then set enableBreakoutSession as false
+          this.updateBreakout(response.body);
+          this.doToggleBreakout(enable);
```

Comment:
I think the promise returned by `toggleBreakout()` should only be resolved once all the async operations started by this function resolve, so here the return value of `this.doToggleBreakout()` should not be ignored and also in line 278.
This function right now has a mix of `async/await` and `.then()` - this is not recommended. Please change it to use await everywhere, so remove the `then()` call and use await when calling `this.doToggleBreakout()` (in both places)